docs/integration_tests.md

```diff
@@ -0,0 +1,103 @@
+# Running integration tests manually
+
+The integration tests ensure HADDOCK3 workflows work properly by
+performing short docking simulations on benchmark systems. With these
```

Comment:
Not everything is docking and these are not benchmark systems

Reply:
for docking :+1:
Okay, I can reword the "benchmark". But a *benchmark* is anything used as a reference. These are benchmarks because they are used as test cases. But I will reword it because HADDOCK uses "benchmark" for a very specific purpose. I think you were referring to that. :+1: